redwood/web/src/layouts/AppLayout/TestnetWarning.tsx

```diff
@@ -0,0 +1,25 @@
+import {Alert, Link, Text} from '@chakra-ui/react'
+
+const chain = process.env.CHAIN_DEPLOYMENT
+
+const TestnetWarning = () => {
+  if (chain === 'test') {
+    return (
+      <Alert display="flex" justifyContent="center" status="warning">
+        <Text>
+          This testnet deployment is for testing only. Visit{' '}
+          <Link href="https://zorro.xyz" fontWeight="bold" colorScheme="black">
+            zorro.xyz
+          </Link>{' '}
+          to access the live protocol.
```

Comment:
- we probably want to avoid giving the impression that there is a live protocol (since there isn't yet),
- we probably want to explicitly let people know that their registration will be erased, etc

Maybe: "This is an alpha testnet deployment of Zorro which is only for testing purposes. Registrations may be reset periodically."